Makefile

```diff
@@ -23,7 +23,13 @@ run-mock-registry: ${GOPATH}/bin/mock-registry vendor
 prepare-build: install
 	cp "${GOPATH}"/bin/broker build/broker
 
-build: prepare-build
+dev-image: prepare-build
+	docker build -f ${BUILD_DIR}/Dockerfile-src ${BUILD_DIR} -t ${BROKER_IMAGE}:${TAG}
+	@echo
+	@echo "Remember you need to push your image before calling make deploy"
+	@echo "    docker push ${BROKER_IMAGE}:${TAG}"
+
+release-image:
```

Comment:
When we build the release image is the expectation that we're using it locally? My understanding is that it would be published.  If that's the case we could just do a ```docker build ...``` and ```docker push ...```.